global.json

```diff
@@ -2,9 +2,9 @@
   "tools": {
     "dotnet": "2.1.401",
     "vs": {
-      "version": "15.8"
+      "version": "16.0"
```

Comment:
:exclamation: This can't change in the **master** branch prior to a merge from **dev16.0-vs-deps**&rarr;**master** and an upgrade in the minimum supported development version of VS to 16.0. You likely already know this, but calling it out explicitly.

Reply:
Why is that the case? This is just changing the build engine we use.

Reply:
It ties into the integration testing infrastructure, which is a fact I found in #33145 but now I'm not sure _exactly_ how it fits in.

Reply:
Seems strange that this would be related. This is just changing the MSBuild engine, nothing more. Not sure how that would break our ability to run tests.

Reply:
I don't really have any objections to doing the merge. It's logical given we're moving to 2019 being fully required when we take this change. Why for that still puzzles me.